src/main/java/ca/ciccc/Assignment2.java

```diff
@@ -90,9 +137,18 @@ public boolean tripleIncreasingOrder(int[] nums) {
      * @return true if the array contains the same number of odds and even numbers, otherwise false.
      */
     public boolean evenOrOdd(int[] nums){
-        // Todo: your code goes here
+        int evenNum = 0;
+        int oddNum = 0;
 
-        return false;
+        for (int i = 0; i < nums.length; i++) {
+            if (nums[i] % 2 == 0) {
+                evenNum += 1;
+            } else {
+                oddNum += 1;
+            }
+        }
+
+        return evenNum == oddNum;
```

Comment:
You have to return TRUE or FALSE.

Reply:
Some companies/projects require you to return the value explicitly. (depends on the code standard for the project). But here, it's alright!